clickfile.py

```diff
@@ -1265,18 +1265,45 @@ def build(tag):
         sys.exit(command_build.returncode)
 
 
-@k6.command("run", help="Run k6 performance test")
+@k6.command("prepare-accounts", help="Generates users and fund its account balances.")
 @click.option("-n", "--network",required=True, default="local", help="Which network to use for envs assignment")
-@click.option("-s", "--script", required=True, default="./loadtesting/k6/tests/sendNeon.test.js", help="Path to k6 script")
-@click.option("-u", "--users", default=None, required=True, help="Number of users (will be generated before load test run)")
+@click.option("-u", "--users", default=None, required=True, help="Number of users to generate before test run")
 @click.option("-b", "--balance", default=None, required=True, help="Initial balance of each user in Neon")
-@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address")
+@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address if needed")
+@catch_traceback
+def prepare_accounts(network, users, balance, bank_account):
+    network_object = network_manager.get_network_object(network)
+    web3_client = NeonChainWeb3Client(proxy_url=network_object["proxy_url"])
+    faucet = Faucet(faucet_url=network_object['faucet_url'], web3_client=web3_client)
+    account_manager = EthAccounts(web3_client, faucet, bank_account)
+    k6_prepare_accounts(account_manager, users, balance)
+
+
+@k6.command("deploy-erc20", help="Deploy erc20 contract.")
+@click.option("-n", "--network",required=True, default="local", help="Which network to use for envs assignment")
+@click.option("-b", "--balance", default=None, required=True, help="Initial balance of erc20 owner")
+@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address if needed")
 @catch_traceback
-def run(network, script, users, balance, bank_account):
+def deploy_erc20(network, balance, bank_account):
     network_object = network_manager.get_network_object(network)
-    k6_prepare_accounts(network, network_object, users, balance, bank_account)
+    web3_client = NeonChainWeb3Client(proxy_url=network_object["proxy_url"])
+    faucet = Faucet(faucet_url=network_object['faucet_url'], web3_client=web3_client)
+    account_manager = EthAccounts(web3_client, faucet, bank_account)
     
-    print("Running load test scenario...")
+    owner = account_manager.create_account(balance=int(balance))
+    erc20 = deploy_erc20_contract(web3_client, faucet, owner)
+    print(f"ERC20 contract deployed at {erc20.contract.address} with owner {erc20.owner.address}")
+
+
+@k6.command("run", help="Run k6 performance test.")
+@click.option("-n", "--network",required=True, default="local", help="Which network to use for envs assignment")
+@click.option("-s", "--script", required=True, default="./loadtesting/k6/tests/sendNeon.test.js", help="Path to k6 script")
```

Comment:
Lets make default value = to run all possible load tests for proxy ?

Reply:
I've planned to collect all scenarios for proxy in a common one. Then run it taking into account weight of each separate scenario. Do you want to have it in this branch?